pkg/build/builder/docker.go

```diff
@@ -239,7 +251,15 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	}
 
 	envVars := getBuildEnvVars(d.build)
-	newFileData = appendEnvVars(newFileData, envVars)
+	newFileData = appendMetadata(Env, newFileData, envVars)
```

Comment:
Well if I would have two separate methods appendEnvVars and appendLabelVars they would 
look identically except the instruction which you are passing. 
No point to dup. these functions